src/main/kotlin/no/skatteetaten/aurora/boober/service/GitService.kt

```diff
@@ -161,6 +176,34 @@ open class GitService(
         }
     }
 
+    /**
+     * This should be used to prevent unwanted changes from being committed.
+     * @param git Git repository to preform clean operation
+     * @param removeUnstaged will remove changed files
+     * @param removeUntracked will remove new files
+     * @return true if clean up was successfully, false if not.
+     */
+    fun cleanRepo(git: Git, removeUnstaged: Boolean = true, removeUntracked: Boolean = true): Boolean {
```

Comment:
her kalles Git for git mens det i funksjonen commitAndPushChanges kalles for repo. Synes det bør være konsekvent.